web/src/app/main/components/homepage/homepage.component.scss

```diff
@@ -49,6 +49,10 @@
           height: 35px;
         }
 
+        img {
+          width: 24px;
+        }
+
```

Comment:
Effects the profile pictures. Comment and screenshots in PR conversation

Reply:
ok

Reply:
I made some updates to this already